utils/inits/db_setup.go

```diff
@@ -21,15 +21,25 @@ func InitializeDatabase() {
 		log.Fatal(err)
 	}
 	userCollectionExists := false
+	bookingCollectionExists := false
 	for _, collectionName := range collectionNames {
 		if collectionName == "users" {
 			userCollectionExists = true
+		} else if collectionName == "bookings" {
+			bookingCollectionExists = true
 		}
 	}
 	if !userCollectionExists {
 		fmt.Println("User collections does not exist, creating `users` collection with `root` user")
 		CreateRootUser()
 	}
+	if !bookingCollectionExists {
+		fmt.Println("Booking collection does not exist, creating `bookings` collection")
+		err := db.CreateCollection(ctx, "bookings")
+		if err != nil {
+			log.Fatal(err)
+		}
+	}
```

Comment:
why is this check required? there is no need to pre-populate the booking collections? the init db features is used for population of static data only